include/rcutils/time.h

```diff
@@ -100,6 +100,84 @@ RCUTILS_WARN_UNUSED
 rcutils_ret_t
 rcutils_steady_time_now(rcutils_time_point_value_t * now);
 
+/// Return a timepoint as nanoseconds in a string.
+/**
+ * The number is always fixed width, with left padding zeros up to the maximum
+ * number of digits the timepoint can represent.
+ * Right now that is 19 digits (so 19 characters) for a signed 64-bit integer.
+ * Positive timepoint values will have a leading space, whereas negative values
```

Comment:
I understand that the motivation for this is so that positive and negative outputs are aligned, but isn't that only going to be necessary in like 0.1% of use cases? I think the small use case of swapping between negative and positive timestamps and wanting them to be aligned doesn't warrant the extraneous space for 99.9% of use cases (I might be missing something though). Having the default output be `[ 123.456]` is surprising to me.

Reply:
> I understand that the motivation for this is so that positive and negative outputs are aligned, but isn't that only going to be necessary in like 0.1% of use cases?

That's right, I had reservations about it too, but given I made these changes in the first place in order to try and keep a fixed width I was placing extreme value on never having a different string length. However, the case pointed out for floats (variable mantissa) is far more likely and the case I'm optimizing for here should honestly never happen (that's why I originally made the timepoint unsigned 😛).

I can change this.

Reply:
+1 for having the default misaligned for positive/negative numbers, if the misalignment bothers someone they can add an option for it :smile: I know we pad in some cases in ros1 e.g. `[ERROR]` vs [ INFO]` but those are much more likely.